programs/bpf/c/src/invoke/invoke.c

```diff
@@ -480,28 +481,29 @@ extern uint64_t entrypoint(const uint8_t *input) {
                                  signers_seeds, SOL_ARRAY_SIZE(signers_seeds)));
     break;
   }
-  case TEST_INSTRUCTION_DATA_TOO_LARGE: {
-    sol_log("Test instruction data too large");
+  case TEST_MAX_INSTRUCTION_DATA_LEN_EXCEEDED: {
+    sol_log("Test max instruction data len exceeded");
     SolAccountMeta arguments[] = {};
-    uint8_t *data = sol_calloc(1500, 1);
+    uint64_t data_len = 10 * 1024 + 1;
+    uint8_t *data = sol_calloc(data_len, 1);
     const SolInstruction instruction = {accounts[INVOKED_PROGRAM_INDEX].key,
                                         arguments, SOL_ARRAY_SIZE(arguments),
-                                        data, 1500};
+                                        data, data_len};
     const SolSignerSeeds signers_seeds[] = {};
     sol_assert(SUCCESS == sol_invoke_signed(
                               &instruction, accounts, SOL_ARRAY_SIZE(accounts),
                               signers_seeds, SOL_ARRAY_SIZE(signers_seeds)));
 
     break;
   }
-  case TEST_INSTRUCTION_META_TOO_LARGE: {
-    sol_log("Test instruction meta too large");
-    SolAccountMeta *arguments = sol_calloc(40, sizeof(SolAccountMeta));
-    sol_log_64(0, 0, 0, 0, (uint64_t)arguments);
+  case TEST_MAX_INSTRUCTION_ACCOUNTS_EXCEEDED: {
+    sol_log("Test max instruction accounts exceeded");
+    uint64_t accounts_len = 256;
```

Comment:
Magic number, might be using a `const` from a definition somewhere else?

Reply:
Fixed, added constants in `cpi.h`